ui/src/views/Documentation/index.jsx

```diff
@@ -120,6 +120,7 @@ export default class Documentation extends Component {
 
       this.setState({ Page, pageInfo, error: null });
     } catch (error) {
+      console.log(Object.assign({}, error));
```

Comment:
Left over debug?

Reply:
ack, yes.  I'm working on another PR to make debugging docs easier (since right now any error is treated as "not found")